experimental/dds/tree2/src/test/shared-tree/fuzz/operationTypes.ts

```diff
@@ -111,6 +111,20 @@ export interface NodeRangePath {
 	count: number;
 }
 
+export interface DisconnectOp {
+	type: "disconnect";
+	index: number;
+	isObserver: boolean;
```

Comment:
This field doesn't seem to be consumed anywhere. Same for `ReconnectOp.isObserver`.